### PR TITLE
feat: add data-testid properties in the item authoring

### PR DESCRIPTION
**Pull request by AnastasiaArcadia**
**Related to:** https://oat-sa.atlassian.net/browse/AUT-774

**How to test:**
Check that `data-testid` attributes are added to Manage Items, Save, Preview, Add outcome declaration buttons on the Item Authoring page

### CODE REVIEW

File: views/js/qtiCreator/plugins/button.tpl [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<li class="btn-info small {{cssClass}}" title="{{title}}">
+<li class="btn-info small {{cssClass}}" title="{{title}}" {{#if testId}}data-testid="{{testId}}"{{/if}}>
```
Comment (bziondik) [inline views/js/qtiCreator/plugins/button.tpl:1]:
```suggestion
<li class="btn-info small {{cssClass}}" title="{{title}}"{{#if testId}} data-testid="{{testId}}"{{/if}}>
```

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: views/js/qtiCreator/plugins/button.tpl (hunk 1 above)
Comment (AnastasiaArcadia) [reply views/js/qtiCreator/plugins/button.tpl:1]: Do we have any code style rules for *.tpl files?

Comment (bziondik) [reply views/js/qtiCreator/plugins/button.tpl:1]: it is just not to have empty space in the end in case no `data-testid` attribute

**Review by Silvia-loza — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Pull request merged**